docker/README.md

```diff
@@ -25,8 +25,11 @@ Linux host, see sections below for details on Windows and macOS.
 
 ## Scripts
 
-`Dockerfile`'s are located in the directory `docker_files`. Currently, there is only
-one Dockerfile directory, namely `ubuntu2204-lo7.5.9`.
+`Dockerfile`'s are located in the directory `docker_files`. Currently, there are two
+Dockerfile directories, namely `ubuntu2204-lo7.5.9` and `ubuntu2310-dev`.
+The first one can be used to build a Docker image with Ubuntu22.04 and LibreOffice version
+7.5.9. The second, allow you to build LibreOffice from source from a specified git tag
```

Comment:
allow**s**

Reply:
@lisajulia Thanks! I have updated the PR and fixed the typo.